docs/usage/cli.md

```diff
@@ -0,0 +1,66 @@
+# The Pyodide Python CLI
+
+Pyodide includes a python CLI command which is a drop in replacement for the
+native python CLI. For example all the following should work as expected:
+```sh
+python -c "some command"
+python file.py
+python -m some.module
+```
+This is primarily useful for testing and experimentation. It can also be helpful
+to install Pyodide Python packages using uv or pip for bundling purposes.
+
+Execution is not sandboxed, this Pyodide Python has direct access to the native
+file system.
+
+## Platform support
+
+It should work on Linux and macOS. There is currently no Windows support. It
+requires an installation of NodeJS >= 20.
+
+## Installation
```

Comment:
Which approach do you think we should prioritize? I'm familiar with using pyodide-build, so I'd prefer that method to be listed first. On the other hand, the approach using GitHub releases is less user-friendly IMO.